io.openems.edge.io.shelly/src/io/openems/edge/io/shelly/shelly3em/IoShelly3EmImpl.java

```diff
@@ -150,6 +150,7 @@ private void processHttpResult(JsonElement result, Throwable error) {
 		Integer currentL2 = null;
 		Integer currentL3 = null;
 		Boolean hasUpdate = false;
+		Boolean overpower = null;
```

Comment:
StateChannels should be `boolean` instead of `Boolean` (this was wrong with my hasUpdate). Pre-initialize with `false`.